docs/README.md

```diff
@@ -2595,7 +2690,7 @@ The only thing you need to do to deploy a whole new completely-independent copy
 
 You can extend Booster by creating rockets. A rocket is just a node package that implements the public Booster rocket interfaces. You can use them for many things:
 
-1. Extend your infrastructure (Currently, only in AWS): You can write a rocket that adds provider resources to your application stack.
+1. Extend your infrastructure: You can write a rocket that adds provider resources to your application stack.
 2. Runtime extensions (Not yet implemented): Add new annotations and interfaces, which combined with infrastructure extensions, could implement new abstractions on top of highly requested use cases.
```

Comment:
Can we create rockets with the other providers?
In that case, we should also change [this part](https://github.com/boostercloud/booster/tree/main/docs#create-your-own-rocket).

Reply:
No, we cannot at this moment, but the idea is allowing it in the future. Let's keep the mentioned part as it meanwhile. Thanks for the comment